### ISSUE TITLE
vultr coreos still doesn't support TCP_CONG_BBR ?

### ISSUE THREAD

**Issue by luxrck** [coreos/bugs#1804]
Current system: CoreOS beta (1298.3.0)

```
# cat /proc/config.gz | gzip -d | grep TCP

CONFIG_INET_TCP_DIAG=m
CONFIG_TCP_CONG_ADVANCED=y
# CONFIG_TCP_CONG_BIC is not set
CONFIG_TCP_CONG_CUBIC=y
# CONFIG_TCP_CONG_WESTWOOD is not set
# CONFIG_TCP_CONG_HTCP is not set
# CONFIG_TCP_CONG_HSTCP is not set
# CONFIG_TCP_CONG_HYBLA is not set
# CONFIG_TCP_CONG_VEGAS is not set
# CONFIG_TCP_CONG_NV is not set
# CONFIG_TCP_CONG_SCALABLE is not set
# CONFIG_TCP_CONG_LP is not set
# CONFIG_TCP_CONG_VENO is not set
# CONFIG_TCP_CONG_YEAH is not set
# CONFIG_TCP_CONG_ILLINOIS is not set
# CONFIG_TCP_CONG_DCTCP is not set
# CONFIG_TCP_CONG_CDG is not set
CONFIG_DEFAULT_TCP_CONG="cubic"
CONFIG_TCP_MD5SIG=y
CONFIG_NETFILTER_XT_TARGET_TCPMSS=m
CONFIG_NETFILTER_XT_TARGET_TCPOPTSTRIP=m
CONFIG_NETFILTER_XT_MATCH_TCPMSS=m
CONFIG_IP_VS_PROTO_TCP=y
CONFIG_RDS_TCP=m
# CONFIG_NET_TCPPROBE is not set
CONFIG_ISCSI_TCP=m
```
no `bbr` related configs.

```
# modprobe tcp_bbr

modprobe: FATAL: Module tcp_bbr not found.
```

#1719 Enable BBR TCP congestion control
coreos/coreos-overlay#2332 sys-kernel/coreos-modules: enable TCP_CONG_BBR

I'm confused, did coreos currently enable `TCP_CONG_BBR` or not ?

**Comment by bgilbert** [comment]
After #1719 was merged, we rolled back to the 4.8 kernel, which doesn't include `tcp_bbr`.  The module is still enabled in our config, so it should reappear once we move to a newer kernel.

**Comment by bgilbert** [comment]
This should be fixed in the next alpha.
